source/common/router/config_impl.cc

```diff
@@ -65,6 +67,82 @@ Optional<uint64_t> HashPolicyImpl::generateHash(const Http::HeaderMap& headers)
   return hash;
 }
 
+HeaderFormatterSharedPtr RequestHeaderParser::parseInternal(const std::string& format) {
+	std::string variable_name;
```

Comment:
Please run this through `fix_format`, this definitely doesn't comply with the style guide (2 spaces, etc.). CI won't allow this to pass until it's compliant.